packages/manager/modules/sms/src/options/blacklist/index.js

```diff
@@ -0,0 +1,21 @@
+import angular from 'angular';
+import controller from './telecom-sms-options-blacklist.controller';
+import template from './telecom-sms-options-blacklist.html';
+
+const moduleName = 'ovhManagerSmsOptionsBlacklist';
+
+angular.module(moduleName, []).config(($stateProvider) => {
+  $stateProvider.state('sms.service.options.blacklist', {
+    url: '/blacklist',
+    views: {
+      'smsView@sms.service': {
```

Comment:
Use a component instead please :)

Reply:
next time ;-)

Reply:
As you are improving this section,I agree with @FredericEspiau that a nice improvement would be to turn this into a component.

Reply:
Done, with component and routing